examples/exNDH1ser.cpp

```diff
@@ -0,0 +1,281 @@
+//                       MFEM Example NDH1
```

Comment:
I agree, but I did not rename exNDH1.cpp because I expect the editor to rename it to a number, i.e. exNDH1.cpp -> ex##p.cpp and exNDH1ser.cpp -> ex##.cpp, where ## is 23 or 24. It depends on whether another new example gets merged to master first, which I cannot predict.

Reply:
We should rename this example before the merge.